app/api.py

```diff
@@ -1,10 +1,21 @@
+from app import db
 from flask import jsonify
+from flask import request
+from .models import Payment
 from flask.views import MethodView
 
 
 class PaymentResource(MethodView):
-    def get(self):
-        return jsonify(ok=True)
-
     def post(self):
-        pass
+        content = request.json
+        if 'note' not in content:
+            raise KeyError
```

Comment:
Resources should not raise any errors since raising an error crashes the handling thread. We'd rather return appropriate message with return code. Like: 
```py
from http import HTTPStatus


return jsonify(error='some descriptive message why we failed'),  HTTPStatus.BAD_REQUEST
```